interface/github.py

```diff
@@ -0,0 +1,58 @@
+"""Utility classes for interacting with Github API via PyGithub."""
+from github import Github, GithubException
+
+
+class GitBot:
+    """Utility class for interacting with Github API."""
+
+    def __init__(self, github, org):
+        """Initialize bot by creating Github object and get organization."""
+        try:
+            self.github = github
+            self.org = self.github.get_organization(org)
+        except GithubException as e:
+            raise GithubAPIException(e.data)
+
+    def org_add_member(self, username):
+        """Add/update to member with given username to organization."""
+        try:
+            user = self.github.get_user(username)
```

Comment:
I assume this is retrieving the Github ID? It might be better to have this method (and all the other methods) take Github ID, since we're storing Github ID and treating that as the "canonical" representation of a user with regards to Github.

Reply:
mmm ill check the dots to see if that works